kubernetes/chart/values.yaml

```diff
@@ -5,70 +5,96 @@
 replicaCount: 1
 
 image:
-  repository: ghcr.io/someengineering/resoto
+  repositoryFolder: ghcr.io/someengineering
```

Comment:
`repositoryPrefix` maybe? It's not really a folder, just some string in front of the image name.